cmd/eksctl-anywhere/cmd/upgradecluster.go

```diff
@@ -114,10 +133,32 @@ func (uc *upgradeClusterOptions) upgradeCluster(cmd *cobra.Command) error {
 		return err
 	}
 
+	_, err = time.ParseDuration(uc.cpWaitTimeout)
+	if err != nil {
```

Comment:
you can combine these two in a single line

```suggestion
	if _, err := time.ParseDuration(uc.cpWaitTimeout); err != nil {
```